lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>Proč?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Tak jsem to vymazal :-)</p>

Reply:
No ne, třeba to má důvod, to si klidně obhaj :)

Reply:
<p>Se mi už nechtělo ;-)</p>